### ISSUE TITLE
UITests fail to run when static frameworks are included

### ISSUE THREAD

**Issue by paulb777** [CocoaPods/CocoaPods#8261]
* [X] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

🌈

# Report

## What did you do?

1. Create a new Xcode project, using the Single View App iOS template
2. Call the project PodsIssue, pick Swift
3. Make sure to check both "Include Unit Tests" and "Include UI Tests"
4. Run pod init
5. Edit the pod file and add the following pods to the top-level target (under # Pods for PodsIssue):

  pod 'Firebase/Core'

6. pod install
7. Open the workspace and attempt running both unit tests and UI tests

Expected: both to pass
Actual: UI tests fail

## What did you expect to happen?

Both to pass

## What happened instead?

UI tests fail
`PodsIssueUITests-Runner.app (39981) encountered an error (Early unexpected exit, operation never finished bootstrapping - no restart will be attempted. (Underlying error: The test runner failed to load the test bundle. Executable cannot be loaded for some other reason, such as a problem with a library it depends on or a code signature/entitlements mismatch.))`

## CocoaPods Environment

Fails with both Xcode 9.2 and 10.1.

### Stack

```
   CocoaPods : 1.5.3
        Ruby : ruby 2.4.4p296 (2018-03-28 revision 63013) [x86_64-darwin17]
    RubyGems : 2.7.8
        Host : Mac OS X 10.13.6 (17G2112)
       Xcode : 10.1 (10B61)
         Git : git version 2.19.0.605.g01d371f741-goog
Ruby lib dir : /Users/paulbeusterien/.rvm/rubies/ruby-2.4.4/lib
Repositories : cpdc-internal-spec - sso://cpdc-internal/spec @ aec88bc6fda9c5ca99ee985a56b2aa8e72075ca4
               master - https://github.com/CocoaPods/Specs.git @ a4d7a3d30c477bb914724144e16000f5a8a0dd94
```

### Installation Source

```
Executable Path: /Users/paulbeusterien/.rvm/gems/ruby-2.4.4/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.2
cocoapods-plugins     : 1.0.0
cocoapods-rome        : 1.0.1
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.3.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'PodsIssue' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  #use_frameworks!
  pod 'FirebaseCore'

  target 'PodsIssueUITests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```

**Comment by dnkoutso** [comment]
huh interesting @paulb777 want to try for 1.6 or 1.7?

**Comment by paulb777** [comment]
Fixed in 1.6.0.beta.2. Thanks @dnkoutso !
